server/controllers/automationController.js

```diff
@@ -43,16 +43,10 @@ const order = [
  * @return  {array}                       array of automation objects
  */
 async function getAutomationDataFromIds(automationRawQuery, querySettings = {}, options = {}) {
-  const automationIds = await models.sequelize.query(automationRawQuery, { ...querySettings }, options);
-  const allData = await automation.findAll({
+  return await automation.findAll({
```

Comment:
@myrdstom you have removed the functionality for pagination and if you are to check your frontend, the pagination for the number of automations returned for a particular page no longer exists.

Reply:
Ahh... thanks, Noted, lemme look into it